app/src/main/java/com/example/poste/activities/FolderViewActivity.java

```diff
@@ -25,32 +16,16 @@
 import androidx.recyclerview.widget.RecyclerView;
 
 import com.example.poste.PosteApplication;
-import com.example.poste.adapters.FolderAdapter;
 import com.example.poste.adapters.PostAdapter;
 import com.example.poste.R;
-import com.example.poste.api.poste.API;
-import com.example.poste.api.poste.exceptions.APIException;
-import com.example.poste.api.poste.models.Folder;
-import com.example.poste.api.poste.models.FolderAccess;
 import com.example.poste.api.poste.models.Post;
-import com.example.poste.api.poste.models.User;
-
-import java.util.HashMap;
-import java.net.URL;
-import java.util.Objects;
 
 /**
  * The FolderViewActivity class adds functionality to the activity_folder_view.xml layout
```

Comment:
It seems there was a lot of confusion when we picked up the project in understand what activities did what (especially since some of them are named in a way that might imply functionality different from what they actually do) -- should we maybe expand on the class docstrings?

This activity is for viewing posts inside a given folder, correct?

Reply:
I will add a better description for this class